packages/auto-approve/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "auto-approve",
+  "version": "1.0.0",
+  "description": "Approves and tags pull requests that meet user-defined criteria",
+  "private": true,
+  "author": "Google LLC.",
+  "license": "Apache-2.0",
+  "repository": "https://github.com/googleapis/repo-automation-bots.git",
+  "homepage": "https://github.com/googleapis/repo-automation-bots",
+  "bugs": "https://github.com/googleapis/repo-automation-bots/issues",
+  "main": "build/src/app.js",
+  "keywords": [
+    "probot",
+    "github",
+    "probot-app"
+  ],
+  "files": [
+    "build/src"
+  ],
+  "scripts": {
+    "compile": "tsc",
+    "start": "probot run ./build/src/auto-approve",
+    "pretest": "npm run compile",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test",
+    "fix": "gts fix",
+    "lint": "gts check"
+  },
+  "dependencies": {
+    "@octokit/openapi-types": "^5.3.2",
+    "ajv": "^7.1.1",
+    "gcf-utils": "^7.1.3",
+    "jsonschema": "^1.4.0"
```

Comment:
I don't think you need this dependency, since you're using `ajv` for schema validation?